src/views/blocks/tickets/extra-available.php

```diff
@@ -25,16 +26,13 @@
 	return;
 }
 
-/** @var Tribe__Tickets__Tickets_Handler $tickets_handler */
-$tickets_handler = tribe( 'tickets.handler' );
-
-$available = $tickets_handler->get_ticket_max_purchase( $ticket->ID );
+$available = $ticket->available();
 
-if ( -1 === $available ) {
+if ( - 1 === $available ) {
 	return;
 }
 
-$post_id   = $this->get( 'post_id' );
+$post_id = $this->get( 'post_id' );
```

Comment:
:no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $post_id

<sub>hash: f7c969f441b827e1d02310c3e89ace44</sub>